test/test.js

```diff
@@ -38,4 +38,17 @@ describe('index.js', function () {
                 expect(request.post.callCount).to.equal(2);
             });
     });
+
+    it('should create a DA case', function () {
+        const proxy = '';
+        const params = {
+            baseUrl: '',
+            authToken: '',
+            caseData: fs.readFileSync('test/test.json')
```

Comment:
I don't know that much about node, but I think what you should do here is not read this file synchronously.
I think it should be something like:
`
fs.readFile('filename', caseData => {
  const params = {
            baseUrl: '',
            authToken: '',
            caseData
  };
  divTestHarness.createDaCase(params, proxy)
            .then(() => {
                expect(request.post.callCount).to.equal(2);
                done();
            });
});
`

But maybe get a third opinion.